### DISCUSSION TITLE
Alloy

### DISCUSSION THREAD

**Discussion by bsdayo** [selfhst/icons#142] [New Icon Request]
Grafana Alloy

Project repository: https://github.com/grafana/alloy
SVG file: https://github.com/grafana/alloy/blob/main/docs/sources/assets/alloy_icon_orange.svg
Preview: ![Preview](https://github.com/grafana/alloy/raw/refs/heads/main/docs/sources/assets/alloy_icon_orange.svg)

**Comment by shollyethan** [comment]
Grafana Alloy has been added - thanks!
